micropython/ssa/core.py

```
@@ -6,7 +6,31 @@
 from umqtt.simple import MQTTClient
 
-from typing import Any, Dict, List, Callable, Awaitable
+from typing import Any, Dict, List, Tuple, Callable, Awaitable
+
+
+class SSA_Property:
+    def __init__(self, value: Any):
+        self.value = value
+        self.has_event = False
+
+    def set_event(self, name: str, trigger: Callable[[Any], Bool], transform: Callable[[Any], Any] | None):
```

Ruff (0.8.2)

Undefined name `Bool`

(F821)